config/webpack/base.js

```diff
@@ -76,30 +64,36 @@ module.exports = {
         test: /\.(png|jpg|ico)?$/,
         loader: 'url-loader?limit=10000&mimetype=image/png',
       },
-      {
-        test: /\.json$/,
-        loader: 'json-loader'
-      },
-      {
-        test: /\.(csv|tsv)$/,
-        loader: 'csv-loader',
-        options: {
-          dynamicTyping: true,
-          header: true,
-          skipEmptyLines: true
-        }
-      },
     ]
   },
+  optimization: {
+    splitChunks: {
```

Comment:
All these changes related with .babelrc, webpack, etc. were done by @fjcalzado, isn't it?
I start my branch from `refactor` branch.

Reply:
maybe not sure, but let's keep it as it is and move forward